tests/integration/steps_start_chains.go

```diff
@@ -82,7 +82,7 @@ func stepsStartChains(consumerName string, setupTransferChan bool) []Step {
 			state: State{
 				chainID("provi"): ChainState{
 					Proposals: &map[uint]Proposal{
-						1: ConsumerProposal{
+						1: ConsumerAdditionProposal{
```

Comment:
@MSalopek great opportunity to improve these consumer chain addition tests by adding in your new `ConsumerChains: &map[chainID]bool{"consu": true}` state accessor!